specification/storagesync/resource-manager/Microsoft.StorageSync/2017-06-05-preview/storagesync.json

```diff
@@ -0,0 +1,3232 @@
+          },
+          {
+            "name": "syncGroupName",
+            "in": "path",
+            "description": "Name of Sync Group resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serverEndpointName",
+            "in": "path",
+            "description": "Name of Server Endpoint object.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "ServerEndpoint Resource",
+          "Actions"
+        ],
+        "operationId": "ServerEndpointRecall_Post",
+        "description": "Recall a serverendpoint.",
+        "x-ms-examples": {
+          "ServerEndpointRecall_Post": {
+            "$ref": "./examples/ServerEndpointRecall_Post.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "Server Endpoint object",
+            "headers": {
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "202": {
+            "description": "Asynchronuous Operation Status Location",
+            "headers": {
+              "Location": {
+                "description": "Operation Status Location URI",
+                "type": "string"
+              },
+              "x-ms-request-id": {
+                "description": "request id.",
+                "type": "string"
+              },
+              "x-ms-correlation-request-id": {
+                "description": "correlation request id.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error message indicating why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorageSync/storageSyncServices/{storageSyncServiceName}/registeredServers": {
+      "get": {
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "storageSyncServiceName",
+            "in": "path",
+            "description": "Name of Storage Sync Service resource.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "tags": [
+          "RegisteredServer Resource"
+        ],
+        "operationId": "RegisteredServersGet_ListByStorageSyncService",

```

Comment:
Operation ID should be in the format `"group_verb"`. So this should be changed to something like:
`"operationId": "RegisteredServers_ListByStorageSyncService"`

Reply:
resolved